src/components/card/Card.js

```diff
@@ -0,0 +1,28 @@
+import { Box, Button, Typography } from '@mui/material'
+
+import { cardStyles as style } from '~/components/card/card.styles'
+import { Link } from 'react-router-dom'
+
+//TODO add possibility for button to redirect user to the specified url.
+
+const Card = ({ img, title, description, btnText, link }) => {
```

Comment:
give a more specific name to the component

Reply:
Fixed.